<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreJoueur</class>
 <widget class="QDialog" name="FenetreJoueur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Joueur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Nom">
       <property name="text">
        <string>Nom </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_Nom"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Prenom">
       <property name="text">
        <string>Prenom *</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_prenom">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_add">
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_add"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_tel">
       <property name="text">
        <string>Telephone</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_tel"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_mail">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="lineEdit_Email"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Equipe</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="10" column="2">
      <widget class="QLineEdit" name="lineEdit_age"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_age">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Maillot">
       <property name="text">
        <string>Maillot *</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_poste">
       <property name="text">
        <string>Poste *</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_lis">
       <property name="text">
        <string>Licence</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_lis"/>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Nom</tabstop>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>lineEdit_lis</tabstop>
  <tabstop>lineEdit_add</tabstop>
  <tabstop>lineEdit_tel</tabstop>
  <tabstop>lineEdit_Email</tabstop>
  <tabstop>lineEdit_age</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FenetreJoueur</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FenetreJoueur</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
